cmd/edit/cluster/cmd.go

```diff
@@ -230,14 +234,13 @@ func run(cmd *cobra.Command, _ []string) {
 	// Enable interactive mode if no flags have been set
 	if !interactive.Enabled() {
 		changedFlags := false
-		for _, flag := range []string{"expiration-time", "expiration", "private",
```

Comment:
Note for this; all I did was remove `disable-workload-monitoring`, each line change was due to reformatting each line manually